src/platform/testing/e2e/form-tester/index.js

```diff
@@ -31,14 +31,6 @@ const getLogger = (debugMode, testName) => (...params) => {
  *  the form.
  */
 const runTest = async (page, testData, testConfig, userToken, testName) => {
-  // Hide the Foresee overlay
-  await page.evaluateOnNewDocument(() => {
```

Comment:
I'd like to try this again, but for now, it's not working so I didn't want to have two things that accomplished the same goal.